backend/src/db/migrations/20250429203304_certificates-ca-relation-removal.ts

```diff
@@ -0,0 +1,46 @@
+import { Knex } from "knex";
+
+import { TableName } from "../schemas";
+
+export async function up(knex: Knex): Promise<void> {
+  if (await knex.schema.hasTable(TableName.Certificate)) {
+    const hasProjectIdColumn = await knex.schema.hasColumn(TableName.Certificate, "projectId");
+    if (!hasProjectIdColumn) {
+      await knex.transaction(async (trx) => {
```

Comment:
starting a transaction here would be redundant because migrations already run inside a file-level txn